README.md

```diff
@@ -356,6 +356,17 @@ config.coverage :CI do |coverage|
 end
 ```
 
+### Ignoring Files
+
+Teaspoon allows defining files and directories to ignore when performing coverage instrumentation.
+
+The below example will ensure that the files within `/test/javascripts` and `/vendor/assets` are not intrumented by Istanbul and will not show up in the coverage results.
+
+```ruby
+config.coverage_ignored_files = ["/test/javascripts", "/vendor/assets"]
```

Comment:
The configuration option here is `coverage_ignored_files` but these are directories. I wonder if this would be better as just `coverage_ignored`.

For the sake of consistency, we should probably make this configuration more like the `matcher` config:

```ruby
suite.matcher = "{spec/javascripts,app/assets}/**/*_spec.{js,js.coffee,coffee}"
```